CHANGELOG.rst

```diff
@@ -5,8 +5,9 @@ Version History
 v5.24.0
 --------
 
+* Extend LOVE frontend routing system to add SSL and subpath serving `<https://github.com/lsst-ts/LOVE-frontend/pull/487>`_
 * Extend LOVE frontend routing system to add SSL and subpath serving `<https://github.com/lsst-ts/LOVE-frontend/pull/485>`_
-* Extend getEFDLogs method to specify the timestamps scale `<https://github.com/lsst-ts/LOVE-frontend/pull/484>`_
```

Comment:
I think you remove the wrong PR jejejeje

Reply:
Oops! Fixing it now. Thanks!